cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -267,6 +258,22 @@
     } );
 }
 
+.m-global-search {
+    // Tab trigger is used to capture press of the tab key so that
+    // global search can be collapsed when it hits this element.
```

Comment:
Just so I'm clear, the user tabs through the entire menu, would normally tab to the next item after the menu, but instead hits this hidden tab-trigger, which has an event observer that in turn closes the menu?